decode/neuralfitter/train/live_engine.py

```diff
@@ -81,23 +81,28 @@ def live_engine_setup(param_file: str, cuda_ix: int = None, debug: bool = False,
 
     """Experiment ID"""
     if not debug:
-        experiment_id = datetime.datetime.now().strftime("%Y-%m-%d_%H-%M-%S") + '_' + socket.gethostname()
-
-        if log_comment:
-            experiment_id = experiment_id + '_' + log_comment
-
+        if param.InOut.checkpoint_init is None:
+            experiment_id = datetime.datetime.now().strftime("%Y-%m-%d_%H-%M-%S") + '_' + socket.gethostname()
+            from_ckpt = False
+            if log_comment:
+                experiment_id = experiment_id + '_' + log_comment
+        else:
+            from_ckpt = True
+            experiment_id = param.InOut.checkpoint.split('/')[-1]
     else:
         experiment_id = 'debug'
+        from_ckpt = False
 
     """Set up unique folder for experiment"""
     experiment_path = Path(param.InOut.experiment_out) / Path(experiment_id)
     if not experiment_path.parent.exists():
         experiment_path.parent.mkdir()
 
-    if debug:
-        experiment_path.mkdir(exist_ok=True)
-    else:
-        experiment_path.mkdir(exist_ok=False)
+    if not from_ckpt:
+        if debug:
+            experiment_path.mkdir(exist_ok=True)
+        else:
+            experiment_path.mkdir(exist_ok=False)
 
     model_out = experiment_path / Path('model.pt')
     ckpt_path = model_out.parent / Path('ckpt.pt')
```

Comment:
ckpt_path should not be hard coded to ckpt.pt relative to the model.
Ckpt_path should be inferred from param.InOut.checkpoint_init (will probably require a bit of logic here).